ecs-init/docker/docker_test.go

```diff
@@ -482,6 +482,13 @@ func TestGetInstanceConfig(t *testing.T) {
 	defer mockCtrl.Finish()
 
 	envFile := "\nECS_ENABLE_GPU_SUPPORT=true\n"
+	defer func() {
+		MatchFilePatternForGPU = FilePatternMatchForGPU
```

Comment:
NIt: to me, MatchFilePattern sounds like a regex pattern alone. I'd rather call it FilepathGlob or something like that.

Reply:
we are infact doing a regex pattern check for `/dev/nvidia*`